### DISCUSSION TITLE
Increase sensitivity in chord recognition

### DISCUSSION THREAD

**Discussion by niklas1102** [CPJKU/madmom#468] [General]
Hi there. 
I've asked this question before in Madmom's "GoogleGroups". But I wanted to ask it again here and also ask here especially @fdlm . It would be really great if you could help me there. Thank you very much in advance.

I use the chord recognition method of Madmom. 
Like this:

```
dcp = DeepChromaProcessor()
decode = DeepChromaChordRecognitionProcessor()
chordrec = SequentialProcessor([dcp, decode])
chroma = dcp("song.wav")
```

Now I wanted to ask for a way to change the sensitivity of this method. So that means that for songs where filtering out chords is particularly difficult, the chords will be filtered out "more easily".

I have a few songs where the filtering works only with about every 3rd chord so for these it would be great to have such a funtion.
Or maybe there is some other solution for my problem?

Many thanks again

**Comment by fdlm** [comment]
Hi, I'm not sure I follow. What do you mean by "filtering out chords"?

**Reply by niklas1102** [reply]
Hi, thanks for your answer.
I probably expressed myself wrong. Sorry. I just want to recognize chords in an audio file using the "DeepChromaChordRecognitionProcessor"-function from Madmom (this function is also described exactly [here](https://madmom.readthedocs.io/en/latest/modules/features/chords.html)) 
And now I want to change the "sensitivity" of this function. I don't know if this is possible. The problem is that with some audio files "too many" chords are recognized, and with others "too few". Is there any solution for this?

Thanks again.

**Reply by fdlm** [reply]
By "too many"/"too few", do you mean that the chords switch too often/not often enough? If you can provide some examples, that would be great. Ideally, chord sequences with "too many"/"too few" chords, and what you would like them to look like.

Generally speaking, since both the `DeepChromaProcessor` and the `DeepChromaChordRecognitionProcessor` are learned from data, it's hard to change a single parameter in order to affect the detected chords in a specific way.
